backend/yandex/yandex_crawl.py

```diff
@@ -0,0 +1,49 @@
+import requests
+from bs4 import BeautifulSoup
+
+
+def to_dict(index, sauce, img_sauce):
+    table = {
+         "img_link": img_sauce,
+         "source": sauce,
+         "host": "Yandex",
+         "tittle": "yandex"+str(index)
+         }
+    return table
+
+
+def spider(max_pic_num):
+    url = "https://yandex.com/images/search?text=skin%20cancer%20dermoscopy"
+    source_code = requests.get(url)
+    plain_text = source_code.text
+    soup = BeautifulSoup(plain_text, "html.parser")
+    counter = 1
+    data_list = []
+    for link in soup.findAll('a', {"class": "serp-item__link"}):
+        # get link
+        href = link.get('href')
+        # getimage link
+        string = href.split("img_url=")[1]
+        convert = string.replace("%2F", "/")
+        convert2 = convert.replace("%3A", ":")
+        og_link = convert2.split('&text')[0]
+        # get source
+        newstring = string.split('%3A%2F%2F')[1]
+        source = newstring.split('%2F')[0]
+        # adding dict
+        data = {}
+        data['Image source'] = source
+        data['Host'] = "Yandex"
+        data['Origin URL'] = og_link
+        data['Title'] = "Yandex" + str(counter)
```

Comment:
![image](https://user-images.githubusercontent.com/38489776/100030178-7e25b480-2e25-11eb-8258-a2e279dc04aa.png)

`Image source` should be the url to direct image.
`Original URL` should be the url where the image can be found.
`Title` should be sth like this
![image](https://user-images.githubusercontent.com/38489776/100030388-f8eecf80-2e25-11eb-8973-0b899f85ff3b.png)

Reply:
Also you should add `https://` for links that don't have it for more secured connections.

Reply:
There's no way I can extract the tittle from the page, the div class that the soup can crawl does not contain the class of the tittle

Reply:
@Huy-Ngo I have inspected the elements of yandex when image search. It's impossible to get the title like that unless request a new response, is it okay for Quang just give the title "Yandex + <number>" ?

Reply:
The elements on wikipedia aren't class'ed either and he could still get it. If you can't get by class, get by the order of the element.

Reply:
Also, if you don't like that approach, you can visit the website and take the website's title (it's in `<title>` tag)

Reply:
@quanghaopro gentle ping

Reply:
@Huy-Ngo so I tried to run @quanghaopro again
![image](https://user-images.githubusercontent.com/38489776/100266279-22be0880-2f84-11eb-806d-85cd4272ce5d.png)
He finally got titltes